isaric/parsers/isaric-ccpuk.toml

```diff
@@ -1077,6 +1083,34 @@
   date = { field = "daily_dsstdat", description = "Date of assessment" }
   context = ['Highest']
 
+[[observation]]
+  name = "fio2_percent"
+  phase = "study"
+  date = { field = "daily_dsstdat", description = "Date of assessment" }
+  value = { field = "daily_fio2b_lborres", apply = { function = "Percentage" } }
+
+[[observation]]
+  name = "fio2_percent"
+  phase = "study"
+  date = { field = "daily_dsstdat", description = "Date of assessment" }
+  value = { field = "daily_fio2_lborres", apply = { function = "Percentage" } }
+
+[[observation]]
+  name = "oxygen_flow_rate"
+  phase = "study"
+  date = { field = "daily_dsstdat", description = "Date of assessment" }
+  value = { field = "daily_fio2c_lborres" }
+
+  [observation.context]
+    combinedType = "set"
+    excludeWhen = "none"
+    fields = [
+      { field = "daily_nasaloxy_cmtrt", values = { 1 = "HFNC" } },
+      { field = "daily_noninvasive_prtrt", values = { 1 = "non-invasive" } },
+      { field = "daily_invasive_prtrt", values = { 1 = "invasive" } },
+      { field = "daily_ecmo_prtrt", values = { 1 = "ECMO" } },
+    ]
```

Comment:
i think it might be best to drop the context values here as I don't think the variables are explicitly related and we don't know whether the fio2 value is related to one of these interventions